checkout-pr

```diff
@@ -26,14 +30,60 @@ else
     PR_NUMBER="$1"
 fi
 
+# which repo?  metron or metron-bro-plugin-kafka
```

Comment:
You should list the options here

Reply:
I considered it, but wasn't sure how to cleanly list them without going to a multi-line menu.  Let me give it a shot.